core/market/decentralized/src/matcher.rs

```diff
@@ -17,22 +17,27 @@ mod error;
 mod resolver;
 mod store;
 
+/// Stores proposal generated from resolver.
+#[derive(Debug)]
+pub struct DraftProposal {
```

Comment:
I will change this into `RawProposal` in my PR #333 
because `Draft` in the context of `Proposal` is misleading. Resulting `Proposal` will be in the `Initial` state

Reply:
Ok